### DISCUSSION TITLE
Why is Bromite sending requests to https://www.bromite.org?

### DISCUSSION THREAD

**Discussion by ygz213** [bromite/bromite#1195] [Q&A] [answered]
I saw https://www.bromite.org in DNS logs after browsing with Bromite. Nothing changed in the app after I blacklisted this address.

**Comment by chromer030** [comment, marked as answer]
This is because of filters used by Bromite AdBlock engine , Bromite frequently checks for new version of this filters which is located in this address :
https://www.bromite.org/filters/filters.dat

And by blacklisting this address , bromite will not be able to download new filters.
Filters list are located here :
https://github.com/bromite/filters

You can also use the extended filter list from this repo , and are very up-to-date
https://github.com/xarantolus/filtrite

**Reply by csagan5** [reply]
It will reach that website if you have any related tile in the NTP, in order to load the favicon.

> Bromite frequently checks for new version

Every week, see https://github.com/bromite/bromite/wiki/AdBlocking

> You can also use the extended filter list from this repo , and are very up-to-date
> https://github.com/xarantolus/filtrite

This is not the official one, obviously.

**Reply by uazo** [reply]
> Every week, see https://github.com/bromite/bromite/wiki/AdBlocking

sorry if i disagree, at each startup, until we recover https://github.com/bromite/bromite/pull/905

**Reply by csagan5** [reply]
The wiki page mentions both; it will connect at startup only if you have previously closed the app.
